lib/main_common.pm

```diff
@@ -174,6 +174,8 @@ sub setup_env {
         set_var('INSTLANG', 'en_US');
     }
 
+    set_var('LTP_KNOWN_ISSUES', 'https://raw.githubusercontent.com/openSUSE/kernel-qe/main/ltp_known_issues.yaml') if is_opensuse and !get_var('LTP_KNOWN_ISSUES');
```

Comment:
```suggestion
    set_var('LTP_KNOWN_ISSUES', get_var('LTP_KNOWN_ISSUES', 'https://raw.githubusercontent.com/openSUSE/kernel-qe/main/ltp_known_issues.yaml')) if is_opensuse;
```

Reply:
The original will fix empty string. Your suggested version will not.

Reply:
That could lead to setting variable to the same value and redundant `bmwqemu::save_vars()` call. Thus, if you don't mind I'd prefer the original code.

Reply:
But what if we don't want to use ltp known issues at all? It will not be possible with this  line.